### PR TITLE
Kill old vnc connection and forward a port for novnc as well

### PR DESCRIPTION
**Pull request by hikalium**

### CODE REVIEW

**Comment by hikalium** [comment]
@mhiramat Please take a look again...! :pray:

**Review by mhiramat — COMMENTED** [review]
I think it is better to add a mode switch option of novnc and kmsvnc.

File: src/cmd/dut.rs [hunk 1]
```diff
@@ -121,29 +121,35 @@ struct ArgsVnc {
...
-    let mut shown = false;
+    let vnc_port = args.vnc_port.unwrap_or(5900);
+    let web_port = args.vnc_port.unwrap_or(6080);
+
+    if let Err(e) = target.run_cmd_piped(&["kill -9 $(pgrep --full vnc)"]) {
+        eprintln!("Failed to kill previous vnc instance: {e}")
+    }
+
+    let mut child_kmsvnc = target.start_port_forwarding(vnc_port, 5900, "kmsvnc")?;
+    let mut child_novnc = target.start_port_forwarding(web_port, 6080, "novnc")?;
```
[the remark this answers is not in the document]
Comment (mhiramat) [reply src/cmd/dut.rs:140]: OK, then let both of them enabled.

File: src/cmd/dut.rs [hunk 2]
```diff
@@ -121,29 +121,35 @@ struct ArgsVnc {
...
+
+    eprintln!("To use VNC via web browser, please open:");
+    eprintln!("  http://localhost:{vnc_port}/vnc.html");
```
[the remark this answers is not in the document]
Comment (mhiramat) [reply src/cmd/dut.rs:143]: And again, this is not "{vnc_port}", but "{web_port}", right?

[the remark this answers is not in the document]
Comment (hikalium) [reply src/cmd/dut.rs]: Fixed! sorry for missing the suggestion... please take a look again!

**Comment by mhiramat** [comment]
Looks good to me :)

**Review by mhiramat — APPROVED** [review]
LGTM.

**Pull request merged**